src/schemas/json/host.json

```diff
@@ -26,10 +26,10 @@
     "functions": {
       "description": "The list of functions the host should load.",
       "type": "array",
-      "items": {
-        "type": "string",
-        "uniqueItems": true
-      }
+			"uniqueItems": true,
+			"items": {
```

Comment:
@GerryFerdinandus  - this still shows following warning on build

 ``` Warning: Error: Unexpected rules in type "string" at #/items Use --force to continue.```

Any idea how to fix this?

Reply:
I see two more of this wrong "uniqueItems": true

see:
"watchDirectories"
"excludedHosts"

Reply:
done!